src/ruuvitag_ble/parser.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import annotations
+
+import logging
+
+from bluetooth_sensor_state_data import BluetoothData
+from home_assistant_bluetooth import BluetoothServiceInfo
+from sensor_state_data import DeviceClass, Units
+
+from ruuvitag_ble.df5_decoder import DataFormat5Decoder
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class RuuvitagBluetoothDeviceData(BluetoothData):
+    """Data for Ruuvitag BLE sensors."""
+
+    def _start_update(self, service_info: BluetoothServiceInfo) -> None:
+        try:
+            raw_data = service_info.manufacturer_data[0x0499]
+        except (KeyError, IndexError):
+            _LOGGER.debug("Manufacturer ID 0x0499 not found in data")
+            return None
+
+        data_format = raw_data[0]
+        if data_format != 0x05:
```

Comment:
Any plans on adding older versions? (not required, just a tip). 

See e.g. https://github.com/custom-components/ble_monitor/blob/master/custom_components/ble_monitor/ble_parser/ruuvitag.py

Reply:
If required; I wanted to go YAGNI here.

I suspect Ruuvi enthusiasts tend to update their tags to the newest firmware and data version anyway.

Reply:
ok,